### ISSUE TITLE
Study how to fix issue: Command in Runtime updated only after restarting workspace

### ISSUE THREAD

**Issue by skabashnyuk** [eclipse-che/che#12588]
### Description

According to @sleshchenko 's quick study of the issue https://github.com/eclipse/che/issues/12333 Command in Runtime updated only after restarting workspace - there is no quick and easy solution.
The goal of this task is to describe the current situation, propose a possible solution and decided with stakeholders how to fix https://github.com/eclipse/che/issues/12333.

**Comment by skabashnyuk** [comment]
won't fix
